src/shared/components/lazyMobXTable/LazyMobXTable.tsx

```diff
@@ -208,29 +213,33 @@ class LazyMobXTableStore<T> {
     @computed public get downloadData()
```

Comment:
Hi guys, I think we should pass a function for downloading.  Formatting the data for download seems very specific to the nature of the particular data being displayed and can't be generalized here.   @adamabeshouse would be a good person to discuss this with.   Is it important that the downloaded data reflect ordering and filtering state or can we just deliver it as a dump given that someone will most likely be using another tool to manipulate the data?